TAScheduler/Managment/LabManagement.py

```diff
@@ -15,10 +31,45 @@ def __init__(self, authID=""):
 	# Lab TA(in) -TA of the lab
     def createLab(self, labName, labHours, labLocation, labDays, labTA):
         pass
+=======
+    # Side-effects: Lab is created and added inside the database
+    #Lab ID(in) - Id of the lab
+    # Lab Name(in) - Name of the lab
+    # Lab Hours(in) - Hours of the lab
+    # Lab Location(in) - Location of the lab
+    # Lab Days(in) - Days of the lab
+    # Lab Instructor(in) - Instructor of the lab
+    # Lab TA(in) -TA of the lab
+    def createLab(request, Id, labName, labLocation, labHours, labDays, labInstructor, labTA):
+        if(not(isinstance(Id, int))):
+            raise TypeError("Id entered is not of type int")
+        if(not(isinstance(labName, str))):
+            raise TypeError("Lab Name entered is not of type str")
+        if(not(isinstance(labLocation, str))):
+            raise TypeError("Lab Location entered is not of type str")
+        if(not(isinstance(labHours, str))):
+            raise TypeError("Lab Hours entered is not of type str")
+        if(not(isinstance(labDays, str))):
+            raise TypeError("Lab Days entered is not of type str")
+        if(not(isinstance(labInstructor, User))):
+            raise TypeError("Lab Instructor entered is not of type User")
+        if(labInstructor.getType() != "INSTRUCTOR"):
```

Comment:
```suggestion
        if(labInstructor.userType != "INSTRUCTOR"):
```
A function called getType() does not exist.